### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by th3defender** [Checkmarx/kics#5381] [Ideas] [answered]
Let me know how to purchase a licensed version

**Comment by kaplanlior** [comment, marked as answer]
Hi @th3defender ,

Welcome to GitHub and to KICS.

This is an Open Source project created by [Checkmarx](https://checkmarx.com/) and released under the [Apache 2.0 License](https://github.com/Checkmarx/kics/blob/master/LICENSE).
